addons/project/static/src/components/project_status_with_color_selection/project_status_with_color_selection_field.js

```diff
@@ -20,6 +20,16 @@ export class ProjectStatusWithColorSelectionField extends SelectionField {
         return this.colors[value] ? this.colorPrefix + this.colors[value] : "";
     }
 }
+
+SelectionField.props = {
+    ...SelectionField.props,
+    status_label: { type: String, optional: true },
+};
+
+ProjectStatusWithColorSelectionField.extractProps = ({ attrs }) =>{
+    return {...SelectionField.extractProps({attrs}), status_label: attrs.status_label};
```

Comment:
```suggestion
    return {...SelectionField.extractProps({attrs}), statusLabel: attrs.status_label};
```
And this is clearly a detail, but don't you think props should be named in `camelCase` (whereas attrs are ok in `snake_case`)? :thinking: